extension/src/testReportProvider.ts

```diff
@@ -0,0 +1,57 @@
+import { TestResourceManager } from "./testResourceManager";
+import { TestSuite, TestLevel } from "./protocols";
+import { TextDocumentContentProvider, Uri } from "vscode";
+
+import * as os from 'os';

```

Comment:
Should add eslint files for checking styles

Reply:
already added tslint.json at the end of last release, however I didn't make it too strict as it introduced many errors. might need to open a separate pr to update tslint.json and fix all style errors.